docs/source/plugin/advanced.rst

```diff
@@ -64,3 +64,115 @@ a QUIT/PART event, and they will be available after a JOIN event.
 
 Note that you don't specifically need to use ``@plugin.thread(False)``, but
 it is still recommended to prevent any race condition.
+
+
+Capability Negociation
+======================
+
+Capability negociation is a feature of IRCv3 that allows a server to advertise
+a list of optional capabilities, and allows its clients to request such
+capabilities. You can see that as feature flags, activated by the client.
+
+Capability negociation takes place after:
+
+* connecting to the IRC server
+* client's identification (``USER`` and ``NICK``)
+
+And before:
+
+* the ``RPL_WELCOME`` event (001)
+* ``ISUPPORT`` messages
+* client's authentification (except for SASL, which occurs in the capability
+  negociation phase)
```

Comment:
```suggestion
* client's authentication (except for SASL, which occurs in the capability
  negotiation phase)
```

'authentification' _is_ a word (despite my browser's spell-checker flagging it), but its meaning is not what's meant here.